prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -116,7 +116,13 @@ class Hl7Serializer(val metadata: Metadata) {
             }
 
             Element.Type.CODE -> setCodeComponent(terser, value, pathSpec, element.valueSet)
-            Element.Type.TELEPHONE -> setTelephoneComponent(terser, value, pathSpec, element)
+            Element.Type.TELEPHONE -> {
+                if (value.isNotEmpty()) {
+                    setTelephoneComponent(terser, value, pathSpec, element)
+                } else {
+                    terser.set(pathSpec, "") // Not at all sure what to do here.
+                }
```

Comment:
This was a bug fix, where the HL7 generation didn't handle an empty phone number field.     The "not sure" comment is just because I don' know whether an empty/blank value at that location will cause trouble for downstream systems.